src/main/resources/application.yaml

```diff
@@ -83,7 +83,12 @@ payment-mode:
       endpoint: "bulk_connector_{MODE}-{dfspid}"
 
 application:
-  bucket-name: paymenthub-ee-dev
+  bucket-name: paymenthub-ee
```

Comment:
please check whether this field is added to the Helm chart and using global variable

Reply:
will address this as part of GOV-75